packages/backend/src/schema.mjs

```diff
@@ -38,5 +38,17 @@ export default [
       ['data', 'String'],
     ],
   },
+  {
+    name: 'AttesterDescription',
+    primaryKey: '_id',
+    rows: [
+      {
+        name: '_id',
+        type: 'String',
+        default: () => nanoid(),
+      },
+      ['data', 'String'],
```

Comment:
I think the schema could be
```ts
{
    name: 'AttesterDescription',
    primaryKey: 'id',
    rows: [
        ['id', String],
        ['network', 'String'], // or chainID
        ['attesterId', 'String'],
        ['data', 'String'],
    ],
},
```

and `id` can be `network + attesterId`

Reply:
fixed